template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/oozie/OozieRoleConfigProvider.java

```diff
@@ -56,4 +71,16 @@ public String getServiceType() {
     protected DatabaseType dbType() {
         return DatabaseType.OOZIE;
     }
+
+    public static boolean isOozieHA(TemplatePreparationObject source) {
+        String cdhVersion = StringUtils.defaultString(source.getBlueprintView().getProcessor().getStackVersion());
+
+        if (isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERA_STACK_VERSION_7_2_11)) {
+            return source.getHostGroupsWithComponent(OozieRoles.OOZIE_SERVER)
+                .filter(hg -> hg.getNodeCount() > 1)
```

Comment:
Shouldn't this be `>=1` ? What if I have 2 Host groups, each with a single Node, but both of them have an Oozie installed? In this case we'd have 2 total Oozie instances meaning Oozie is in HA.

Reply:
That type of configuration is not supported. It should fail with load-balancer not configured error message if the customer themselves are not adding load-balancers via custom templates. I do not know how common such a setup is. If the customers raise a feature request then I need to re-architect the target groups for the load-balancers to be spread across multiple host groups. In Azure it is strictly checked to be in a single host group at the moment because we are still using availability sets.